.swiftlint.yml

```diff
@@ -6,6 +6,7 @@ disabled_rules:
   - opening_brace
   - valid_docs
   - type_name
+  - vertical_whitespace
```

Comment:
I didn't have the latest `swiftlint` so violations did not show in my environment. Looking at those files, I don't think that multiple whitespaces help readability, so I would rather keep default settings for rule and remove whitespaces.

As for tests - it is IMO better to disable `function_length` rule then disable linting altogether (even Quick templates have it disabled by default 😃). I've created issue #197